Formula/cmockery.rb

```diff
@@ -3,7 +3,9 @@ class Cmockery < Formula
   homepage "https://github.com/google/cmockery"
   url "https://storage.googleapis.com/google-code-archive-downloads/v2/code.google.com/cmockery/cmockery-0.1.2.tar.gz"
   sha256 "b9e04bfbeb45ceee9b6107aa5db671c53683a992082ed2828295e83dc84a8486"
-  license "BSD-3-Clause"
+  # Installed COPYING is BSD-3-Clause but source code uses Apache-2.0.
+  # TODO: Change license to Apache-2.0 on next version as COPYING was replaced by LICENSE.txt
+  license all_of: ["BSD-3-Clause", "Apache-2.0"]
```

Comment:
From tarball:
- src/cmockery.c
    ```
     * Copyright 2008 Google Inc.
     *
     * Licensed under the Apache License, Version 2.0 (the "License");
     * you may not use this file except in compliance with the License.
     * You may obtain a copy of the License at
     * 
     * http://www.apache.org/licenses/LICENSE-2.0
    ```
- COPYING
    ```
    Copyright (c) 2006, Google Inc.
    All rights reserved.
    
    Redistribution and use in source and binary forms, with or without
    modification, are permitted provided that the following conditions are
    met:
    
        * Redistributions of source code must retain the above copyright
    notice, this list of conditions and the following disclaimer.
        * Redistributions in binary form must reproduce the above
    copyright notice, this list of conditions and the following disclaimer
    in the documentation and/or other materials provided with the
    distribution.
        * Neither the name of Google Inc. nor the names of its
    contributors may be used to endorse or promote products derived from
    this software without specific prior written permission.
    
    THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
    ```

Adding `Apache-2.0` avoids having separate audit exception and matches state of most code/upstream.